go/vt/vttablet/onlineddl/schema.go

```diff
@@ -139,6 +139,12 @@ const (
 		WHERE
 			migration_uuid=%a
 	`
+	sqlUpdateInOrderCompletionPendingCount = `UPDATE _vt.schema_migrations
+			SET in_order_completion_pending_count=%a
+		WHERE
+			migration_uuid=%a AND
+			in_order_completion_pending_count!=%a
+	`
```

Comment:
Superfluous, the result will be the same, MySQL knows there's no change anyway. This query has near zero impact as it is, and simpler is better.
```suggestion
	sqlUpdateInOrderCompletionPendingCount = `UPDATE _vt.schema_migrations
			SET in_order_completion_pending_count=%a
		WHERE
			migration_uuid=%a
	`
```